tinystr/src/helpers.rs

```diff
@@ -0,0 +1,22 @@
+use std::num::NonZeroU32;
+use std::ptr::copy_nonoverlapping;
+
+use super::Error;
+
+pub(crate) unsafe fn make_4byte_str(
+    text: &str,
+    len: usize,
+    mask: u32,
+) -> Result<NonZeroU32, Error> {
+    // Mask is always supplied as little-endian.
+    let mask = mask.to_le();
```

Comment:
An incredibly minor nit, but this should be `from_le`. The semantics are the same, but it communicates the intent of converting from a little-endian to a native-endian representation.